mappings/1.8#1.8.1-pre1.tinydiff

```diff
@@ -214,14 +214,16 @@ c	net/minecraft/C_24qhtkqyz$1	1
 	m	(Ljava/lang/Object;)Z	apply	apply	
 		p	1		p_1	
 c	net/minecraft/C_25obpltwe
-	f	D	f_84dysutwo		f_84dysutwo
-	f	[D	f_41tptzxcz		f_41tptzxcz
-	f	[D	f_42sxgxbfu		f_42sxgxbfu
-	m	(DD)D	m_43fwsbzrl		m_43fwsbzrl
-		p	0			p_0
-		p	2			p_2
-	m	(D)D	m_97siwddxn		m_97siwddxn
-		p	0			p_0
+	f	D	f_84dysutwo		MAGIC_ATAN2_CONSTANT
+	f	[D	f_41tptzxcz		ASIN_TABLE
+	f	[D	f_42sxgxbfu		X_TO_Y_TABLE
```

Comment:
This table stores `cos(arcsin(a))` right? That would be converting y to x on the unit circle.